packages/kit/src/routes/config/getStateFromPath.ext.ts

```
@@ -88,29 +99,16 @@ type ParsedRoute = {
 
-  // ---CHANGED Begin----: support hash router
-  path = globalThis.location.hash.split('#').pop() || '/';
-  // ---CHANGED end----
-
-  if (options) {
-    validatePathConfig(options);
-  }
-
-  let initialRoutes: InitialRouteConfig[] = [];
+   // ---CHANGED Begin----: support hash router
+   path = globalThis.location.hash.split('#').pop() || '/';
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)